shared/js/angularProjects.js

```diff
@@ -107,27 +107,27 @@ controllers.controller('ProjectsController', ['$scope', '$http', '$route', '$rou
   };
 
   $scope.showPublishProjectDialog = function (project) {
-    window.onShowConfirmPublishDialog(publishProject.bind(this, project));
+    var projectType = getProjectType(project);
```

Comment:
Is there any reason this code should still live in sharedjs now that we can output JS that goes through our pipeline and is used by both dashboard and pegasus?